README.md

```diff
@@ -136,7 +133,7 @@ Please install a TensorFlow on your compiling machine, The compiler needs to kno
 its headers according to the installed TensorFlow. 
 
 ```
-export TF_VERSION=["2.5.1", "2.4.1", "1.15.2"]
+export TF_VERSION="2.5.1"  # "2.5.1", "2.4.1", "1.15.2" are valid.
```

Comment:
Maybe are verified is more broadly fit for users with different TF versions, when compiling from source.